lib/frontend/src/frontend.rs

```diff
@@ -546,6 +547,59 @@ impl<'a> FunctionBuilder<'a> {
     }
 }
 
+/// Helper functions
+impl<'a> FunctionBuilder<'a> {
+    /// Calls libc.memcpy
+    ///
+    /// Copies the `size` bytes from `src` to `dest`, assumes that `src + size`
+    /// won't overlap onto `dest`.
+    pub fn call_memcpy(&mut self, dest: Value, src: Value, size: Value) {
```

Comment:
Could you also add a `memmove` version for overlapping memory, or add a bool for it to this function?